crates/fuel-core/src/state.rs

```diff
@@ -1,9 +1,6 @@
-use crate::{
-    database::{
-        Column,
-        Result as DatabaseResult,
-    },
-    state::in_memory::transaction::MemoryTransactionView,
+use crate::database::{
+    Column,
```

Comment:
This state module was suppose to be something that could be extracted from the main fuel-core crate without any knowledge of its' data model. Why do we need to include the columns here?

Reply:
It was here before=)
![Uploading image.png…]()

Reply:
I see, nm then. But maybe something to resolve in the future.